### ISSUE TITLE
[Feature Request]: History of assets at location

### ISSUE THREAD

**Issue by devvonmonahan** [grokability/snipe-it#12495]
### Is your feature request related to a problem? Please describe.

We would like the ability to see all assets that have been at a location before (this includes assets assigned to a user that is based in that location, and assets assigned directly to the location).

The main purpose for this is to track back in time which assets may have been there, say, a year ago.
Particularly useful if you think you are missing assets and want to know where they ended up.

### Describe the solution you'd like

Being able to click on the location page and have a nice display of assets that have been at that location now, and bonus points for a column that says their current status/location/assignee - without having to click on each asset.

### Describe alternatives you've considered

Maybe using the API

**Comment by SoreThumb1** [comment]
This is what I was also looking for. Having a "History" tab for locations, similar to the one on asset and user pages would be perfect.

![image](https://github.com/snipe/snipe-it/assets/25835158/0bd047b2-db9d-4d6b-b73e-3eb6679e9656)

**Comment by nandorusrin** [comment]
Yeah, I need this and maybe for company too. I need to know the item that goes in/out from a company to company or location to location or if being deleted.

**Comment by grga911** [comment]
This feature would be perfect, also adding option to se changes on location fields (name, phone, e-mail etc.).
